wkcuber/Dockerfile

```
@@ -29,4 +29,4 @@ COPY cluster_tools/README.md /cluster_tools/
 RUN poetry config virtualenvs.create false --local
 RUN poetry install --no-dev
 
-ENTRYPOINT [ "python", "-m" ]
+ENTRYPOINT [ "python", "-m" ]
```

```suggestion
ENTRYPOINT [ "python", "-m" ]

```

We usually keep the trailing newlines in.